src/components/Home/Home.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import leftPhotoURL from "../../assets/sample-photo.jpg";
+import logoOrange from "../../assets/logo-orange.png";
+import styled from 'styled-components';
+import devices from '../../utility/MediaQueries.js';
+
+const Outer = styled.div`
+  width: 100%;
+  margin-top: 80px;
+  height: calc(100vh - 80px);
+  align-items: center;
+`
+
+const Left = styled.div`
+  position: absolute;
+  background-color: #fff;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+  padding-left: 3em;
+  height: calc(100vh - 80px);
+  width: calc(67vw - 3em);
+  ${devices.small`
+    width: 100vw;
+  `}
+  color: rgb(241,93,36);
+  left: 0;
+  overflow: scroll;
+`
+
+const Right = styled.div`
```

Comment:
this is also a very vague name for the styled component - we should make this more descriptive/specific.